StarTable format specification.md

```diff
@@ -142,8 +142,8 @@ functionality. The different block types are summarized in this table:
 
 | Block type    | Start marker first-column cell content                       | End marker                                             | Description & remarks                                        |
 | ------------- | ------------------------------------------------------------ | ------------------------------------------------------ | ------------------------------------------------------------ |
-| Directive     | `***` followed by directive block name                       | - Empty first column cell; or<br>- New block start     | Placeholder for a variety of functions e.g.  <br> - Version control <br> - Allowing tables from various sources to be added dynamically to the set of tables statically present in a sheet |
-| Table         | `**`  followed by table name                                               (which may not start with `*`) | - Empty first column cell; or <br> - New block start   | Primary data content                                         |
+| Directive     | `***` followed by directive block name (which may not contain `*`) | - Empty first column cell; or<br>- New block start     | Placeholder for a variety of functions e.g.  <br> - Version control <br> - Allowing tables from various sources to be added dynamically to the set of tables statically present in a sheet |
+| Table         | `**`  followed by table name                                               (which may not contain `*`) | - Empty first column cell; or <br> - New block start   | Primary data content                                         |
 | Template      | Starts with `:`                                              | -   Empty first column cell; or <br> - New block start | Template data embedded in template files allow input files to be matched against a template, and provide a description of input data. |
 | Metadata line | Ends with `:`, and is not a valid start marker for one of the other block types | End of line                                            | Are only accepted at the top of a sheet, before any other block types.<br />Provide information about the current sheet.<br />Always span exactly one line. |
 
```

Comment:
For the "layer" concept to make sense, the block level should completely describe how to split a file into blocks.
From this point of view, transposed tables should either be a block type on it's own (probably messy), or mention that a table name may be decorated.